run.sh

```diff
@@ -1,7 +1,14 @@
 #!/bin/sh
-set -eu
+set -e
```

Comment:
You can keep `-u` and evaluate variables that can be possible unset like this: `${JAVA_HOME:-}`